### DISCUSSION TITLE
Mali G90T

### DISCUSSION THREAD

**Discussion by Loganavter** [skyline-emu/skyline#2066] [Q&A] [answered]
Will it work on Realme 6 8/128?

**Comment by Mononono87** [comment, marked as answer]
It probably will but the performance won't be that good.
I'm saying most 3d games won't run and some 2d games might struggle

**Reply by Loganavter** [reply]
![Screenshot_2022-12-10-06-03-00-59_64ef5fc2000c1caa954c114bb372e1d5](https://user-images.githubusercontent.com/66126148/206825825-30cc36ff-41f7-4e9d-add5-66d0828a20f7.jpg)
Custom drivers are not available on all mali processors?

**Reply by Mononono87** [reply]
They are not

**Reply by Loganavter** [reply]
?

**Reply by Mononono87** [reply]
Mali does not support custom drives
